### PR TITLE
Fixed reports' headers

### PR DESCRIPTION
**Pull request by dorsha**
fixes: https://github.com/demisto/server/issues/6603

### CODE REVIEW

File: Reports/report-last24HoursClosedIncidentsReport.json [hunk 1]
```diff
@@ -384,7 +384,17 @@
...
+        "tableColumns": ["Name", "Occurred", "Type", "Owner", "Severity", "Status", "Closed", "Close notes"],
+        "readableHeaders": {
+          "name": "Name",
+          "occurred": "Occurred",
+          "type": "Type",
+          "owner": "Owner",
+          "severity": "Severity",
+          "status": "Status",
+          "closed": "Closed",
+          "closeNotes": "Close notes"
```
Comment (aviadl) [inline Reports/report-last24HoursClosedIncidentsReport.json]: Upper case N ?

File: Reports/report-last24HoursIncidentReport.json [hunk 2]
```diff
@@ -411,7 +411,16 @@
...
-        "tableColumns": ["name", "occurred", "type", "owner", "severity", "status", "dueDate" ],
+        "tableColumns": ["Name", "Occurred", "Type", "Owner", "Severity", "Status", "Due date" ],
+        "readableHeaders": {
+          "name": "Name",
+          "occurred": "Occurred",
+          "type": "Type",
+          "owner": "Owner",
+          "severity": "Severity",
+          "status": "Status",
+          "dueDate": "Due date"
```
Comment (aviadl) [inline Reports/report-last24HoursIncidentReport.json]: same for Due Date ?

File: Reports/report-last24HoursClosedIncidentsReport.json (hunk 1 above)
Comment (dorsha) [inline Reports/report-last24HoursClosedIncidentsReport.json]: no.. ido and dan said that only the first word should start with uppercase.

File: Reports/report-last24HoursIncidentReport.json (hunk 2 above)
Comment (dorsha) [inline Reports/report-last24HoursIncidentReport.json]: no.. ido and dan said that only the first word should start with uppercase.

**Comment by dorsha** [comment]
The other reports will be in another PRs.
And we don't support it from version in report.. we will need to provide new build.

**Comment by aviadl** [comment]
We will need to add this fromVersion, toVersion so we will not have problems on the next versions we will enhance those reports ...
Since this mens that those reports will not work on any old env ...

**Comment by dorsha** [comment]
OK, I have no idea what to change in the server for that.. need to provide new build

**Comment by dorsha** [comment]
Added two new reports from neelima.

File: Reports/report-MTTRbyIncidentType2Quar.json [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+        }
+      },
+      "layout": {
+        "rowPos": 7,
+        "columnPos": 1,
+        "tableColumns": ["Month Closed", "Type", "# Incidents", "Mean Time to Close(days)"],
+        "classes": "striped stackable small very compact",
+        "readableHeaders": {
+          "type": "Type",
+          "closed(m)": "Month Closed",
```
Comment (aviadl) [inline Reports/report-MTTRbyIncidentType2Quar.json]: you said lower c :)

File: Reports/report-MTTRbyIncidentType2Quar.json [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+          "avg|openDuration / (3600*24)": "Mean Time to Close(days)",
```
Comment (aviadl) [inline Reports/report-MTTRbyIncidentType2Quar.json]: same here, also missing a space

File: Reports/report-MTTRbyOwner2Quar.json [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+          }
+        }
+      },
+      "layout": {
+        "rowPos": 7,
+        "columnPos": 1,
+        "tableColumns": [ "Owner", "Month Closed", "Total Incidents", "Mean Time to Close(days)"],
+        "classes": "striped stackable small very compact",
+        "readableHeaders": {
+          "owner": "Owner",
```
Comment (aviadl) [inline Reports/report-MTTRbyOwner2Quar.json]: Capitalization issues here as wll

File: Reports/report-MTTRbyIncidentType2Quar.json (hunk 3 above)
Comment (dorsha) [inline Reports/report-MTTRbyIncidentType2Quar.json]: It is not mine

File: Reports/report-MTTRbyIncidentType2Quar.json (hunk 4 above)
Comment (dorsha) [inline Reports/report-MTTRbyIncidentType2Quar.json]: not mine as well

File: Reports/report-MTTRbyOwner2Quar.json (hunk 5 above)
Comment (dorsha) [inline Reports/report-MTTRbyOwner2Quar.json]: not mine as well.. will fix

**Comment by dorsha** [comment]
Fixed issue. 
And added `"fromversion": "2.5.0"` to the reports.

**Comment by dorsha** [comment]
Fixed all comments, added from version 2.5.0, and uploaded two new reports.
Merging.

**Pull request merged**